### PR TITLE
Update plugin.json

### PR DESCRIPTION
**Pull request by ornithes**

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -10,4 +10,6 @@
         { "hook": "filter:navigation.available", "method": "getNavigation"}
     ],
     "templates": "static/templates"
-}
\ No newline at end of file
+    "languages": "languages",
+	"defaultLang": "en_GB"
```
Comment (julianlam) [inline plugin.json:14]: Adjust indentation to match please :smile: 

Can you merge all changes into a single pull request?

**Comment by barisusakli** [comment]
Don't we already have these keys in core?

**Comment by aribuga** [comment]
Excuse me, I'm a little confused. I sent piece by piece.

File: plugin.json (hunk 1 above)
Comment (aribuga) [inline plugin.json:14]: of course

**Comment by aribuga** [comment]
@barisusakli https://github.com/NodeBB/NodeBB/blob/v1.x.x/public/language/en_GB/recent.json

Here we can get?

**Pull request closed without merging**